tests/unit/infrastructure/test_migrated_lru_cache_replacements.py

```
@@ -0,0 +1,100 @@
+
+from tunacode.infrastructure.cache import clear_all
+
+
+@pytest.fixture
+def clean_caches() -> None:
+    clear_all()
+    yield
+    clear_all()
+
+
+def test_models_registry_cache_clears_via_clear_all(clean_caches: None) -> None:
```

Ruff (0.14.14)

[warning] Unused function argument: `clean_caches`

(ARG001)